### PR TITLE
Do not use extra starttime process field on Linux

### PR DESCRIPTION
**Pull request by cgzones**
Closes: #242

### CODE REVIEW

**Review by BenBE — APPROVED** [review]
LGTM.

File: unsupported/UnsupportedProcessList.c [hunk 1]
```diff
@@ -56,8 +56,8 @@ void ProcessList_goThroughEntries(ProcessList* super) {
     proc->priority = 0;
     proc->nice = 0;
     proc->nlwp = 1;
-    strncpy(proc->starttime_show, "Jun 01 ", sizeof(proc->starttime_show));
     proc->starttime_ctime = 1433116800; // Jun 01, 2015
```
Comment (BenBE) [inline unsupported/UnsupportedProcessList.c:59]: What's so magic about this date? Shouldn't this be either automagically be updated or some value we can detect elsewhere?

Comment (cgzones) [reply unsupported/UnsupportedProcessList.c:59]: This is for (yet) unsupported platforms. Thus we don't know how to get the starttime of a process.
I don't know what's special about Jun 1, maybe a birthday?

Comment (BenBE) [reply unsupported/UnsupportedProcessList.c:59]: I'd prefer to have 0 here, so some easy "unknown value" check could be done.

**Pull request merged**